chameleonultragui/macos/Runner/AppDelegate.swift

```diff
@@ -1,9 +1,13 @@
 import Cocoa
 import FlutterMacOS
 
-@NSApplicationMain
+@main
```

Comment:
Same here? Did apple just change smthng again?

Reply:
It's generated by a newer version of flutter, see https://github.com/flutter/flutter/pull/146848
Both `@NSApplicationMain` and `@main` is OK here